src/main/java/tdl/record_upload/RecordAndUploadApp.java

```diff
@@ -163,6 +175,35 @@ public static void main(String[] args)  {
         }
     }
 
+    private static void checkDiskspaceRequirements(long minimumRequiredDiskspaceHumanReadable) {
+        log.info("Checking diskspace");
+        long minimumRequiredDiskspace = minimumRequiredDiskspaceHumanReadable * ONE_GB;
+        String userDirectory = System.getProperty("user.dir");
+        String userDriveOrVolume = Paths.get(userDirectory).getRoot().toString();
+        long availableDiskspace = getAvailableDiskspaceFor(userDriveOrVolume);
+        long availableDiskspaceInGB = availableDiskspace / ONE_GB;
+        float availableDiskspaceInMB = (float) (availableDiskspace / ONE_MB);
+        log.info(String.format("Available disk space on the volume (or drive) '%s': %dGB (%.3fMB)", userDriveOrVolume, availableDiskspaceInGB, availableDiskspaceInMB));
+        if (availableDiskspace < minimumRequiredDiskspace) {
+            log.error(String.format("Sorry, you need at least %dGB of free disk space on this volume (or drive), in order to run the screen recording app (in either modes: video-enabled or video-disabled).", minimumRequiredDiskspaceHumanReadable));
```

Comment:
We should not mention the "video-enabled or video-disabled" modes as some of our clients require us to enforce the recording.

Reply:
Fixed